README.md

```diff
@@ -1,6 +1,48 @@
 # Easier68k [![Build Status](https://travis-ci.org/Chris-Johnston/Easier68k.svg?branch=master)](https://travis-ci.org/Chris-Johnston/Easier68k)
 
-A python package based assmebler, disassembler and simulator of the Motorola 68k CPU Architecture.
+Easier68k is a python library that assembles to 68k and simulates the Motorola 68k CPU Architecture.
```

Comment:
Easier68k is a python library that assembles ~~to 68k~~ and simulates the Motorola 68k CPU Architecture.